src/ebay-tri-state-checkbox/__tests__/index.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useRef, ChangeEvent } from 'react'
+import { StoryFn, Meta } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import { EbayLabel } from '../../ebay-field'
+import { EbayTriStateCheckbox } from '../index'
+
+const meta: Meta<typeof EbayTriStateCheckbox> = {
+    component: EbayTriStateCheckbox,
+    title: 'form input/ebay-tri-state-checkbox',
+    argTypes: {
+        checked: {
+            options: ["false", "mixed", "true"],
+            type: { category: "Options" },
```

Comment:
It should be:

```
control: { type: 'select'}
```

Make sure that you are able to make changes in the storybook controls section